### PR TITLE
#163213618 Fixes errors given on feedback

### PR DESCRIPTION
**Pull request by emdeechege**
### What does this PR do?
This pull request Fixes errors given on feedback

### Description of tasks to be completed
The following are the tasks to be completed
* Consistent use of docstrings
* Rename test files to show they cover the endpoints
* Removes commented out code
* Correct use of imports in test files
### Pivotal tracker story associated with this PR:
https://www.pivotaltracker.com/n/projects/2235156/stories/163213618

### CODE REVIEW

**Comment by bencyn** [comment]
looks great

File: app/api/v1/models/questions_models.py [hunk 1]
```diff
@@ -24,38 +26,24 @@ def post_question(self, postedBy, meetup_id, title, content):
...
-            downvote = {
-                "postedBy": question[0]["postedBy"],
-                "meetup_id": question[0]["meetup_id"],
-                "title": question[0]["title"],
-                "content": question[0]["content"],
-                "votes": self.votes - 1
-            }
-
-            return downvote, {"message": "downvote successfull"}
+    def upvotes(self,question_id):
```
Comment (khwilo) [inline app/api/v1/models/questions_models.py:37]: The method upvotes() and downvotes() contain a duplicate implementation except for the part where the votes are either increased or decreased.

**Review by khwilo — COMMENTED** [review]
Consider refactoring your methods where there is a duplicate implementation.

File: app/api/v1/models/questions_models.py (hunk 1 above)
Comment (emdeechege) [reply app/api/v1/models/questions_models.py:37]: @kwhilo, noted with thanks. I intend to add a function in the validation.py to counter the same as soon as this PR is approved.

**Comment by khwilo** [comment]
Before pushing your branch to build on Travis CI run your tests to see the errors you have encountered and try to fix them.

**Pull request merged**